readme.md

```diff
@@ -68,13 +68,13 @@ $ ava --init
 
 ```json
 {
-  "name": "awesome-package",
-  "scripts": {
-    "test": "ava"
-  },
-  "devDependencies": {
-    "ava": "^0.11.0"
-  }
+	"name": "awesome-package",
```

Comment:
This was part of the merge conflict (this part was an old version from whenever I made my fork). Fixed when I merged readme from master.